test/src/test/java/hudson/model/DirectoryBrowserSupportTest.java

```diff
@@ -60,6 +62,7 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
```

Comment:
Why not remove this entirely? Also in the other edited files, and move completely to hamcrest.